bin/hipcc.pl

```diff
@@ -175,6 +175,9 @@ BEGIN
 
         # escape possible spaces in path name
         $HIPCC =~ s/\s/\\$&/g;
+        
+        # put hipcc path in quotes on windows
```

Comment:
I don't think this is needed here, as we escape spaces in the path name in the above line - but I'm not sure if it "hurts" to add it.  We use the HIPCC variable in a command statement below (using backticks), to get the HIP_CLANG_RT_LIB.

Reply:
Ah, would one be preferable over the other? My preference goes to quotation marks as that is more readable, both as code and printed strings.